manuscript/ha-docker-swarm/shared-storage-ceph.md

```diff
@@ -155,21 +155,24 @@ Grab the secret associated with the new user (you'll need this for the /etc/fsta
 ceph-authtool /etc/ceph/keyring.dockerswarm -p -n client.dockerswarm
 ```
 
+!!! note
+    These commands need to be executed from within the running `ceph-mon` container. To do this, we will use the `docker exec` command to open a shell *inside* the running container in order to run the commands above:
+    ```docker exec -it ceph-mon /bin/sh``` 
+
 ### Mount MDS volume
 
 On each node, create a mountpoint for the data, by running ```mkdir /var/data```, add an entry to fstab to ensure the volume is auto-mounted on boot, and ensure the volume is actually _mounted_ if there's a network / boot delay getting access to the gluster volume:
 
 ```
 mkdir /var/data
 
-MYHOST=`hostname -s`
-echo -e "
+echo " \
 # Mount cephfs volume \n
-$MYHOST:6789:/      /var/data/      ceph      \
```

Comment:
Removed the MYHOST as the MDS service binds itself to the LAN IP address, whereas the MYHOST value resolves to 127.0.0.1 causing the mount to time out.

Reply:
Since we're improving this, should we comment that it's possible (and preferable, even) to do this?

```
<node1_ip>,<node2_ip>,<node3_ip>:6789/       /var/data/      ceph      \
```